src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,81 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
Do it like this please:
List<String> dataFromFile = readFile(fromFileName);
String report = createReport(dataFromFile);
writeFile(report, toFileName);